cmd/desync/cat.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/folbricht/desync"
+	"io"
+)
+
+const catUsage = `desync cat [options] <caibx>
```

Comment:
This one allows an optional output file, should it say `desync cat [options] <caibx> [<outputfile>]` ?